iStore/Home/HomeViewModel.swift

```diff
@@ -28,4 +48,16 @@ final class HomeViewModel: ObservableObject {
             }
         }
     }
+    
+    func showProducts(completion: @escaping (ProductsData) -> Void) {
```

Comment:
Should be private?

Reply:
can't be private as It's accessed from the HomeView.

getProduct() & getFilteredProduct should be private.